skipper.go

```diff
@@ -277,6 +277,10 @@ type Options struct {
 	// will apply changes to all matching routes.
 	EditRoute *eskip.Editor
 
+	// A custom routing pre processor implementation that will
+	// be applied to all routes.
+	CustomRoutingPreProcessor routing.PreProcessor
```

Comment:
I am wondering should we just use a slice
```go
RoutingPreProcessors []routing.PreProcessor
```
and init `ro.PreProcessors` with it (or with a copy) to make it generic.
Otherwise we would need to invent a new name for the second custom pre-processor.

Reply:
This would be similar to `CustomFilters`.

Reply:
Hello @AlexanderYastrebov , thx for the quick reply! That makes sense, I'll adjust it to a list of `PreProcessors`.